source/reporting/index.html.md.erb

```diff
@@ -0,0 +1,113 @@
+select __Download all transaction details as a CSV file__. 
+
+You can process CSV files using spreadsheet software.
+
+## Reporting via the GOV.UK Pay API 
+
+Before reading this section, you should read the section describing [Making
+payments](/making_payments). 
+
+You can use the GOV.UK Pay API to export data automatically into your finance
+systems. This requires you to do some development, but could remove 
+routine manual operations from your workflow and lead to significant cost
+savings for your organisation. Before you do this, you should have a datastore 
+where you are already recording payments. 
+
+You can use the GOV.UK Pay API to:
+
+* get information about single payments
+* generate a list of payments 
+
+## Get information about single payments
+
+### Check the status of a single payment
+
+`GET /v1/payments/paymentId`
+
+1. Sign in to <a href="https://gds-payments.gelato.io/api-explorer/"
+   target="blank">the API Explorer</a> (link opens in new window).  
+2. Under __Resource__, select __{Payment Id}. Under __Action__, select <a
+   href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/general/"
+   target="blank">__Find payment by ID__</a> (link opens in new window). 
+   
+For a valid request, the JSON response body contains a `"state"` field. 
+
+For example, for a successful payment:
+
+```
+  "state": {
+    "status": "success",
+    "finished": true
+    },
+```
+
+### See information users have entered for a single payment
+
+`GET /v1/payments/paymentId`
+
+1. Sign in to <a href="https://gds-payments.gelato.io/api-explorer/"
+   target="blank">the API Explorer</a> (link opens in new window).  
+2. Under __Resource__, select __{Payment Id}. Under __Action__, select <a
+   href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/general/"
+   target="blank">__Find payment by ID__</a> (link opens in new window). 
+
+For a valid request, the JSON response body contains the `"card_details"`
+section. This shows information that users enter when making a payment. 
+
+For example:
+
+```
+  "card_details": {
+    "last_digits_card_number": "1111",
+    "first_digits_card_number": "100002",
+    "cardholder_name": "Sherlock Holmes",
+    "expiry_date": "12/21",
+    "billing_address": {
+      "line1": "221 Baker Street",
+      "line2": "Flat b",
+      "postcode": "NW16XE",
+      "city": "London",
+      "country": "GB"
+    },
+```
+
+## Generate a list of payments (search payments) 
+
+`GET /v1/payments`
+
+1. Sign in to <a href="https://gds-payments.gelato.io/api-explorer/"
+   target="blank">the API Explorer</a> (link opens in new window).  
+2. Under __Resource__, select __General__. Under __Action__, select <a
+   href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/general/"
+   target="blank">__Search payments__</a> (link opens in new window). 
+
+You can search payments using the following criteria:
```

Comment:
We may want to mention that all the criteria for a payment must match to be included in the returned list

Reply:
Ah OK, I didn't know that. So users have to input _all_ criteria to get a return?

Reply:
I meant that if your search criteria specifies, say, the reference and the last four digits of the card number, a payment will only be returned if both the reference _and_ the last four digits match (it’s an “and” search, not an “or” search).

Reply:
Gotcha, will amend, thanks.